app/src/window.d.ts

```diff
@@ -0,0 +1,12 @@
+export {};
```

Comment:
```suggestion
```

Reply:
Earlier I had empty export there to treat it as a module, because when I removed it, I got this error Property 'Config' does not exist on type 'Window & typeof globalThis'.ts(2339). So, if we export defaults from here, it maybe automatically resolved.

Reply:
Gotcha, yup. I encountered the same thing.